src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java

```diff
@@ -199,4 +199,11 @@ public void selectAllNewEntries() {
             }
         }
     }
+
+    public void selectAllEntries() {
+        unselectAll();
+        for (BibEntry entry : entriesListView.getItems()) {
+            entriesListView.getCheckModel().check(entry);
+        }
```

Comment:
Doesn't `entriesListView.getCheckModel().checkAll();` work here?

Reply:
You're right, thanks! Updated the code to use .checkAll().